### ISSUE TITLE
Having issues reproducing test error statistics when configuring model.

### ISSUE THREAD

**Issue by pavel-jahoda-sewts** [DequanWang/tent#21]
I've trained my model and have some tests statistics X. Afterward, I an untented, but configured version of the model and I am unable to reach the accuracy X. I've changed the batch size to match the training batch size, but still the issue persist. It probably has to do with using the "model.train()" function. Note, I am using Efficientnet_v2_s as my model.
